guidelines/index.html

```diff
@@ -28,7 +28,7 @@ <h3>Background on WCAG 2</h3>
 				<p>Web Content Accessibility Guidelines (WCAG) 2.2 defines how to make Web content more accessible to people with disabilities. Accessibility involves a wide range of disabilities, including visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities. Although these guidelines cover a wide range of issues, they are not able to address the needs of people with all types, degrees, and combinations of disability. These guidelines also make Web content more usable by older individuals with changing abilities due to aging and often improve usability for users in general.</p>
 				<p>WCAG 2.2 is developed through the <a href="https://www.w3.org/WAI/standards-guidelines/w3c-process/">W3C process</a> in cooperation with individuals and organizations around the world, with a goal of providing a shared standard for Web content accessibility that meets the needs of individuals, organizations, and governments internationally. WCAG 2.2 builds on WCAG 2.0 [[WCAG20]] and WCAG 2.1 [[WCAG21]], which in turn built on WCAG 1.0 [[WAI-WEBCONTENT]] and is designed to apply broadly to different Web technologies now and in the future, and to be testable with a combination of automated testing and human evaluation. For an introduction to WCAG, see the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG) Overview</a>.</p>
                 
-				<p>Significant challenges were encountered in defining additional criteria to address cognitive, language, and learning disabilities, including a short timeline for development as well as challenges in reaching consensus on testability, implementability, and international considerations of proposals. Work will carry on in this area in future versions of WCAG. We encourage authors to refer to our supplemental guidance on <a href="https://www.w3.org/WAI/standards-guidelines/wcag/#supplement">improving inclusion for people with disabilities, including learning and cognitive disabilities, people with low-vision, and more</a>.</p>
+				<p>Significant challenges were encountered in defining additional criteria to address cognitive, language, and learning disabilities, including a short timeline for development as well as challenges in reaching consensus on testability, implementability, and international considerations of proposals. Work will carry on in this area in future versions of WCAG. We encourage authors to refer to our <a href="https://www.w3.org/WAI/WCAG2/supplemental/about/">supplemental guidance on improving inclusion</a> for people with disabilities, including <a href="https://www.w3.org/WAI/WCAG2/supplemental/#-cognitive-accessibility-guidance">cognitive and learning disabilities</a>, low vision, and more.</p>
```

Comment:
should that first link go to https://www.w3.org/WAI/WCAG2/supplemental/ rather than the /about subpage?
```suggestion
				<p>Significant challenges were encountered in defining additional criteria to address cognitive, language, and learning disabilities, including a short timeline for development as well as challenges in reaching consensus on testability, implementability, and international considerations of proposals. Work will carry on in this area in future versions of WCAG. We encourage authors to refer to our <a href="https://www.w3.org/WAI/WCAG2/supplemental/">supplemental guidance on improving inclusion</a> for people with disabilities, including <a href="https://www.w3.org/WAI/WCAG2/supplemental/#-cognitive-accessibility-guidance">cognitive and learning disabilities</a>, low vision, and more.</p>
```

Reply:
> should that first link go to https://www.w3.org/WAI/WCAG2/supplemental/ rather than the /about subpage?

I think it's important that readers get the context that is at the top of the About page.
(I feel medium about this, not super strongly.)

Reply:
> and not quite sure why we need to link to COGA specifically in addition

In getting previous versions of WCAG 2 (2.0? 2.1? both? I don't recall) through, there was an agreement that WCAG would link to supplemental guidance on COGA. I was thinking that we would want to highlight the COGA part.

Since COGA is featured prominently in that supplemental guidance (and I'm confident always will be), then I think OK not to have that second link if there are reasons not to.

Reply:
With both links there, it seems to make sense to have one go to the about, otherwise it's all the same page.

There was (and is) and agreement to link to the COGA supplimental guidance, but I think that's taken care of in the "[Layers of guidance](https://www.w3.org/TR/WCAG22/#wcag-2-layers-of-guidance-0)" section in WCAG.

Reply:
So @alastc is this what you are advocating?
```suggestion
				<p>Significant challenges were encountered in defining additional criteria to address cognitive, language, and learning disabilities, including a short timeline for development as well as challenges in reaching consensus on testability, implementability, and international considerations of proposals. Work will carry on in this area in future versions of WCAG. We encourage authors to refer to our <a href="https://www.w3.org/WAI/WCAG2/supplemental/about/">supplemental guidance on improving inclusion</a> for people with disabilities, including cognitive and learning disabilities, low vision, and more.</p>
```

Reply:
Ah, I didn't realize that a link to Content Usable had been added to that section (last paragraph).
Is it too late to fix the sentence structure? (especially since it's about cognitive disabilities, the irony is unfortunate :) @iadawn fyi